client/config/toml.go

```diff
@@ -27,41 +26,42 @@ broadcast-mode = "{{ .ClientConfig.BroadcastMode }}"
 trace = "{{ .ClientConfig.Trace }}"
 `
 
-var configTemplate *template.Template
-
-func InitConfigTemplate() {
-	var err error
-
+// InitConfigTemplate initiates config template that will be used in
+// WriteConfigFile
+func InitConfigTemplate() *template.Template {
```

Comment:
I have replaced Initial implementation of [Init](https://github.com/cyberbono3/cosmos-sdk/blob/cyberbono3/8529-add-client-config/server/config/toml.go#L205) function by InitConfigTemplate. 

In initial implementation there is a global variable [configTemplate](https://github.com/cyberbono3/cosmos-sdk/blob/cyberbono3/8529-add-client-config/server/config/toml.go#L203). I decided to get rid of it in favour of local [variable](https://github.com/cyberbono3/cosmos-sdk/blob/cyberbono3/8529-add-client-config/client/config/config.go#L136). I hope it makes sence.